src/compiler/clang.rs

```diff
@@ -272,7 +272,14 @@ mod test {
             "-debug-info-kind=constructor"
         );
         assert_eq!(
-            ovec!["-Xclang", "-mllvm", "-Xclang", "-instcombine-lower-dbg-declare=0", "-Xclang", "-debug-info-kind=constructor"],
```

Comment:
Did you intend to include this change?  It looks fine but seems unrelated to the rest of the PR.

Reply:
It's caused by "cargo fmt" - I will revert it